acsylla/_cython/result/result.pyx

```diff
@@ -71,8 +80,10 @@ cdef class Result:
         for i in range(self.column_count()):
             error = cass_result_column_name(self.cass_result, i, <const char**> &name, <size_t*> &length)
             raise_if_error(error)
-            columns.append(name[:length].decode())
-        return columns
+            yield name[:length].decode()
```

Comment:
Maybe we already had a bug here, is there any chance that we could  be leaking some memory due to the usage of `cass_result_column_name`?

Reply:
I didn't notice, but I'll check before release anyway